src/node/ext/server.cc

```diff
@@ -242,7 +277,9 @@ NAN_METHOD(Server::TryShutdown) {
     return Nan::ThrowTypeError("tryShutdown can only be called on a Server");
   }
   Server *server = ObjectWrap::Unwrap<Server>(info.This());
+  TryShutdownOp *op = new TryShutdownOp(server, info.This());
```

Comment:
should this `TryShutdown` call wrap the whole wrapper inner grpc server shutdown in a null check as in `ShutdownServer`?